### DISCUSSION TITLE
G04*

### DISCUSSION THREAD

**Discussion by lookme2** [Argmaster/pygerber#197] [Q&A] [answered]
I have a Gerber file from Altium.  The first few lines look like this.

`G04 #@! TF.GenerationSoftware,Altium Limited,Altium Designer,23.4.1 (23)*
G04*
G04 Layer_Color=65535*
%FSLAX44Y44*%
%MOMM*%
G71*`

The parser errors out on the 2nd G04*

Any ideas?

Frankie

**Comment by Argmaster** [comment]
I bet parser does not expect an empty G04 and that could be the reason. Could you please open an issue for this? From what I see your code is valid according to standard and should be parsed correctly.

**Reply by lookme2** [reply]
Done.  Thanks!

**Comment by Argmaster** [comment, marked as answer]
Fixed in PyGerber 2.3.2

**Reply by lookme2** [reply]
Worked like a champ!  Thanks!!!
